test/e2e/fixtures/directive-require-html/script.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+angular.
+  module('test', []).
```

Comment:
I think it would be better if you overwrote `$exceptionHandler` to put the error message into a DOM element.
Then you could easily read the error in the test (and verify that it is what you expect).

Reply:
Good idea, didn't think of that